lms/templates/sysadmin_dashboard_gitlogs.html

```diff
@@ -93,12 +93,14 @@ <h2 class="instructor-nav">
         </thead>
         <tbody>
           <%
-          if course_id == None:
-              logs = cilset[:10]
-          else:
-              logs = cilset[:2]
+              if not detailed_view:
```

Comment:
nit: it would be clearer to just be switching on `if detailed_view`. slightly harder to parse by dealing with the "not" case first.